java/org.travlyn.server/src/test/java/org/travlyn/server/externalapi/access/OpenRouteStopAccessTest.java

```diff
@@ -0,0 +1,23 @@
+package org.travlyn.server.externalapi.access;
+
+import org.junit.Test;
+import org.junit.jupiter.api.Tag;
+import org.travlyn.shared.model.api.Stop;
+
+@Tag("unit")
+public class OpenRouteStopAccessTest {
+
+    @Test
+    public void testStopCall(){
+        //OpenrouteRequest request = new OpenrouteRequest();
```

Comment:
Souldn't we include this test? Who is responsible for fixing... you or me?

Reply:
Both tests are fixed and improved